packages/data/lib/mappers/cast_mapper.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:data/database/database_model/cast_model.dart';
+import 'package:domain/base/mappers/base_mappers/mapper_base.dart';
+import 'package:domain/model/response_model_people.dart';
+
+class CastMapper
+    extends Mapper<List<Map<String, Object?>>, List<ResponseModelPeople>> {
+  @override
+  List<ResponseModelPeople> call(List<Map<String, Object?>> params) => params
+      .map((json) => CastModel.fromJson(json)).toList()
+      .map((e) => e.toPeopleModel()).toList();
```

Comment:
let's think, do we need additional data transfer model (DB model) in your implementation. Mb better to map object directly to map and reverse?

Reply:
We discussed this with you, we are waiting for further developments :)